### DISCUSSION TITLE
Do we need clearer definition of general terminalogoy in Filecoin?

### DISCUSSION THREAD

**Discussion by askender** [filecoin-project/lotus#7058] [Ideas]
Background info:
I read the doc [Lotus Miner: Splitting main miner and markets service processes](https://docs.filecoin.io/mine/lotus/split-markets-miners/) ,  and found that terminalogoy in Filecoin may need clearer definition.

## An example:
The three words `mining/sealing/proving` are used in this doc, which mean different things.

But  different words `proving`&`mining` are used in the case below, talking about the same thing:
```
lotus-worker
--enable-gpu-proving                     enable use of GPU for mining operations (default: true)
```

In my opinion: 
mining means:  things related to winningpost
proving means:  things related to windowpost
sealing means:  things related  to P1&P2&C1&C2 

So maybe we can use clearer terminalogoy in the command-line of `lotus-worker`.

**Comment by de-authority** [comment]
Yeah. it's too long anyway.

**Comment by dd45e640b42e6da7da96faee3996ef7c** [comment]
yes we do. 

what i have on the list is:

- what is a "pool/mining pool" in fielcoin terms
- what exactly is a "node" in filecoin terms

not having a common, clearly defined language will create confusion.

with the sub process split for the lotus miner software it will need clearer definitions

@askender maybe even move that over to the community repo? or you think we want this discussion based on lotus and force it on the other implementations?
